src/main/java/org/fundacionjala/at15/pokemon/Trainer.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.at15.pokemon;
+
+public class Trainer {
+    private final int limit = 6;
+    private Wallet wallet = new Wallet();
+    private String[] teamPokemon = new String[limit];
+    private int numberOfPokeballs = 0;
```

Comment:
Once we have Pokeball class this could be:
`private ArrayList<Pokeball> pokeballs`
and could be initialized in the class constructor with:
`this.pokeballs = new ArrayList<Potion>();`

Reply:
yes it will be like that